python/app/qt_importer.py

```diff
@@ -41,11 +41,8 @@
         except ImportError:
             pass
 
-    if not imported_qt:
-        # Try PySide
-        from PySide import QtCore, QtGui
```

Comment:
No, please remove the except ImportError: from PySide6 above to get the same behavior if we can't find a PySide interface

```python
    if not imported_qt:
        # Try PySide6
         from PySide6 import QtCore
         import PySide6.QtGui as __PySide_QtGui
         import PySide6.QtWidgets as __PySide_QtWidgets
    else:
        # Patch PySide2/PySide6 to keep the QtGui interface consi
        [...]
```